RHEL6: cleanup sysctl

ybznek opened the pull request:
https://github.com/OpenSCAP/scap-security-guide/issues/1668
```#only-for:...``` in .csv should be probably removed, but I didn't want to change state output DS (https://github.com/OpenSCAP/scap-security-guide/issues/1665)

jan-cerny commented:
Tentatively self-assigning this.

jan-cerny commented on RHEL/6/templates/csv/sysctl_values.csv:
```diff
@@ -19,3 +20,9 @@ net.ipv4.ip_forward,0
 net.ipv6.conf.default.accept_redirects,
+net.ipv6.conf.all.accept_ra,#only-for:oval
+net.ipv6.conf.all.accept_redirects,#only-for:oval
+#net.ipv6.conf.all.disable_ipv6,1
```
Why have you added this and commented it out? I can't find any such rule in current master.

ybznek replied:
Good point, thank you

jan-cerny commented:
Why `sysctl_kernel_dmesg_restrict.xml` is not removed by this PR? Why is it marked as only for bash and ansible in `RHEL/6/templates/csv/sysctl_values.csv` ? Maybe I'm missing something, but I think it should be removed from `/input/oval` and it should not have a special comment in the CSV.

jan-cerny commented on RHEL/6/templates/csv/sysctl_values.csv:
```diff
@@ -1,5 +1,6 @@
+# Add <sysctl_parameter_name, desired_value> to generate hard-coded OVAL and remediation content.
```
In original `input/oval/templates/sysctl_values.csv` there used to be a comment that explained meaning of lines that have no parameter and end with a comma:

`# Add <sysctl_parameter_name,> to generate OVAL and remediation content that use the XCCDF value.`

I suggest moving it here as well, because it can be unclear why some lines have a number and some haven't.

ybznek replied:
Nice catch, thank you.

jan-cerny commented:
@ybznek One more question: I have noticed that in `RHEL/6/input/oval/templates` still remains files `sysctl_ipv6_values_orig.csv` and `sysctl_values_orig.csv`. Do they have any meaning or can they be removed?

ybznek commented:
@jan-cerny  They can be removed. But I expected another removing iterations, so I would remove it later.

ybznek commented:
I've removed commented line in csv, started to generate kernel_dmesg, added missing header into csv.
Unfortunately, I've decided to rebase it.
Thank you for reviewing, @jan-cerny

jan-cerny commented:
Looks good to me. Awesome.
I'll wait only for Jenkins,
